data/mods/Magiclysm/monsters/demon_spider.json

```
@@ -61,7 +61,7 @@
   {
     "id": "mon_demon_spider",
     "type": "MONSTER",
-    "description": "This spider is the size of a car.  Its huge compound eyes seem to be infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of viscious black liquid that sizzles when it touches the ground.",
+    "description": "This spider is the size of a car.  Its huge compound eyes seem to be infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of vicious black liquid that sizzles when it touches the ground.",
```

```suggestion
    "description": "This spider is the size of a car.  Its huge compound eyes look almost like infinite pools of blackness that seem to suck the light out of the air.  Its maroon carapace is studded with wicked-looking spikes that drip with some kind of vicious black liquid that sizzles when it touches the ground.",
```
Stylistic - avoiding "seem" twice in one sentence.